pandas/tests/scalar/period/test_asfreq.py

```diff
@@ -30,9 +30,6 @@ def test_asfreq_near_zero_weekly(self):
         assert week1.asfreq("D", "E") >= per1
         assert week2.asfreq("D", "S") <= per2
 
-    @pytest.mark.xfail(
-        reason="GH#19643 period_helper asfreq functions fail to check for overflows"
-    )
     def test_to_timestamp_out_of_bounds(self):
         # GH#19643, currently gives Timestamp('1754-08-30 22:43:41.128654848')
```

Comment:
you can remove this comment as well I think?

Reply:
updated